.travis.yml

```diff
@@ -7,10 +7,13 @@ cache:
     - $HOME/.composer/cache
 
 php:
-  - 5.6
   - 7
   - 7.1
   - hhvm
 
+matrix:
+    allow_failures:
+        - php: hhvm
```

Comment:
Not the correct solution -  HHVM should simply be installed in a PHP7-compatible version